config/scout.php

```diff
@@ -133,9 +134,9 @@
         'host' => env('MEILISEARCH_HOST', 'http://localhost:7700'),
         'key' => env('MEILISEARCH_KEY'),
         'index-settings' => [
-            // 'users' => [
-            //     'filterableAttributes'=> ['id', 'name', 'email'],
-            // ],
+            'entities' => [
+                'filterableAttributes'=> ['id', 'campaign_id'],
+            ],
```

Comment:
Missing a "docs/meilisearch.md" file to explain how to set things up. How do I batch add characters, locations, posts?